src/main/java/br/com/metricminer2/scm/GitRepository.java

```diff
@@ -100,6 +105,9 @@ public SCMRepository info() {
 			String origin = git.getRepository().getConfig().getString("remote", "origin", "url");
 
 			return new SCMRepository(this, origin, path, headId.getName(), lastCommit.getName());
+		} catch (RepositoryNotFoundException rnfe) {
+			log.error(rnfe.getMessage());
+			return null;
```

Comment:
why this `return null`? If we can't info the repo, I'd say an exception should be thrown.